regolith/schemas.py

```diff
@@ -1168,14 +1169,14 @@
                           'initial project plan',
              'audience': ['lead', 'pi', 'group_members'],
              'status': 'proposed',
-             'type': 'meeting'
+             'type': ALLOWED_MILESTONE_TYPES,
```

Comment:
this is not right here.  see if you can figure out the logic.....  What is the EXEMPLAR doing and what is the SCHEMA doing?  How are they doing it?

Reply:
If I'm being honest, from the file, I have no idea what the difference between the exemplar and the schema is or if they are necessarily separate.

Reply:
Thanks for the honesty.   Maybe a good topic for a convo with one of the other group members?